pyproject.toml

```diff
@@ -14,6 +14,7 @@ python = "^3.10"
 stringcase = "^1.2.0"
 pyyaml = "^6.0"
 pydantic = "^2.4.2"
+pytest = "^8.3.4"
```

Comment:
Please move this to a dev dependency, as pytest isn't required for distribution.

Reply:
Must have missed that; should be fixed nox